src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -226,7 +226,7 @@ def _find_family_capability(sku, supported_families):
         return _get_default_capability(supported_families)
 
 
-def _find_performance_level_capability(sku, supported_service_level_objectives, allow_reset_family):
+def _find_performance_level_capability(sku, supported_service_level_objectives, allow_reset_family, requesting_serverless=False):
```

Comment:
This is not extensible to other compute models. Please just pass in `compute_model` as parameter.

Reply:
make sense, I think that is a good idea. I will update it